### PR TITLE
[9.0][IMP] partner_financial_risk: Improve performance

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Check Travis

**Comment by luismontalba** [comment]
Reviewing this PR on runbot I hesitate about the behaviour of the _Specific limits_.
I set all them to $ 1000 in a customer and create a $ 750 new draft infoice:

seleccion_001
When I validate the invoice the warning pop up but the amount is less than the fixed limit .. 

seleccion_002

Is this correct?

**Comment by andhit-r** [comment]
@LuisMOntalba you should pay attention to Exception message. There are several type of exception. In your case Financial risk exceded means total risk (750) must be greater than credit limit(0). You should adjust the credit limit.

**Comment by luismontalba** [comment]
I did it but I couldn't adjust the credit limit. The credit limit field is editable but resets to 0.00 when saving changes in the customer form forcing exception. I manage Sales Manager permission.
Is it wrong?

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and tested.

File: partner_financial_risk/models/res_partner.py [hunk 1]
```diff
@@ -62,32 +62,54 @@ def _compute_risk_allow_edit(self):
         is_editable = self.env.user.has_group(
             'base.group_sale_manager') or self.env.user.has_group(
             'account.group_account_manager')
-        for partner in self:
+        for partner in self.filtered('customer'):
             partner.risk_allow_edit = is_editable
```
Comment (luismontalba) [inline partner_financial_risk/models/res_partner.py:67]: I'm still worried about "Credit limit" field is not editable when user has sale manager or account adviser rights.

Comment (carlosdauden) [reply partner_financial_risk/models/res_partner.py:67]: Can you tell me the steps to reproduce the problem?

**Review by luismontalba — APPROVED** [review]
IT works !!
(the field was blocked for partner contact changesets)

File: partner_financial_risk/models/res_partner.py (hunk 1 above)
Comment (carlosdauden) [reply partner_financial_risk/models/res_partner.py:67]: The module partner_changeset adds validation to changes in credit_limit field.

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

**Pull request merged**